aldryn_forms/boilerplates/bootstrap3/templates/aldryn_forms/emails/notification.subject.txt

```diff
@@ -1 +1,3 @@
-[Form submission] {{ form_name }} 
\ No newline at end of file
+{% with subject=form_plugin.email_notification_subject %}
+{% if subject %}{{ subject }}{% else %}[Form submission] {{ form_name }}{% endif %}
+{% endwith %}
```

Comment:
I've not encountered an issue with it, based on the raw email data I see the extra spaces have been stripped.
have you encountered an issue related to this?